terraform/README.md

```diff
@@ -0,0 +1,65 @@
+# Grafana-agent-k8s Terraform Module
+
+This Grafana-agent-k8s Terraform module aims to deploy the [grafana-agent-k8s charm](https://charmhub.io/grafana-agent-k8s) via Terraform.
+
+## Getting Started
+
+### Prerequisites
```

Comment:
I think it would be good to mention, that the Juju controller needs to be bootstrapped onto the cluster before we can use this module.

Reply:
https://discourse.charmhub.io/t/set-up-your-development-environment-with-microk8s-for-juju-terraform-provider/13109 link includes steps to prepare the environment (including Juju controller bootstrap) so we did not write it again.

Reply:
Added.